src/modules/ekf2/EKF2.cpp

```diff
@@ -1040,11 +1040,12 @@ void EKF2::PublishBaroBias(const hrt_abstime &timestamp)
 	if (_ekf.aid_src_baro_hgt().timestamp_sample != 0) {
 		const BiasEstimator::status &status = _ekf.getBaroBiasEstimatorStatus();
 
-		if (fabsf(status.bias - _last_baro_bias_published) > 0.001f) {
+		if (fabsf(status.bias - _last_baro_bias_published) > 0.001f || !_baro_bias_published) {
```

Comment:
You could just drop this completely, just limit it to publishing above when the timestamp_sample actually changes. Then just make sure the logging rate is minimal.